### PR TITLE
icu4c 63.1

### PR DESCRIPTION
**Pull request by lembacon**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by lembacon** [comment]
@BrewTestBot test this please

**Comment by fxcoudert** [comment]
- `postgresql` needs a revision bump.
- `couchdb` does not download (error 404 on all mirrors)
- `sile` fails to build with:

```
08:31:43 ==> luarocks make lua-zlib-1.2-0.rockspec --tree=/usr/local/Cellar/sile/0.9.5-alpha_6/libexec/vendor
08:31:43 
08:31:43 Error: Could not find header file for ZLIB
08:31:43   No file zlib.h in /usr/local/include
08:31:43   No file zlib.h in /usr/include
08:31:43   No file zlib.h in /include
08:31:43 You may have to install ZLIB in your system and/or pass ZLIB_DIR or ZLIB_INCDIR to the luarocks command.
08:31:43 Example: luarocks install lua-zlib ZLIB_DIR=/usr/local
```

- `generate-json-schema` test fails with:

```
06:11:26 ==> /usr/local/Cellar/generate-json-schema/2.6.0/bin/generate-schema
06:11:26 internal/fs/utils.js:453
06:11:26     throw err;
06:11:26     ^
06:11:26 
06:11:26 TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be one of type string, Buffer, or URL. Received type object
06:11:26     at Object.open (fs.js:406:3)
06:11:26     at ReadStream.open (internal/fs/streams.js:110:12)
06:11:26     at new ReadStream (internal/fs/streams.js:99:10)
06:11:26     at Object.createReadStream (fs.js:1725:10)
06:11:26     at Object.<anonymous> (/usr/local/Cellar/generate-json-schema/2.6.0/libexec/lib/node_modules/generate-schema/bin/generate-schema:110:15)
06:11:26     at Module._compile (internal/modules/cjs/loader.js:688:30)
06:11:26     at Object.Module._extensions..js (internal/modules/cjs/loader.js:699:10)
06:11:26     at Module.load (internal/modules/cjs/loader.js:598:32)
06:11:26     at tryModuleLoad (internal/modules/cjs/loader.js:537:12)
06:11:26     at Function.Module._load (internal/modules/cjs/loader.js:529:3)
06:11:26 Error: generate-json-schema: failed
06:11:26 An exception occurred within a child process:
06:11:26   Test::Unit::AssertionFailedError: <0> expected but was
06:11:26 <1>.
```

- `mapnik` fails to build with:

```
05:37:49 In file included from plugins/input/sqlite/sqlite_datasource.cpp:23:
05:37:49 In file included from plugins/input/sqlite/sqlite_datasource.hpp:46:
05:37:49 plugins/input/sqlite/sqlite_connection.hpp:171:9: error: use of undeclared identifier 'sqlite3_enable_load_extension'
05:37:49         sqlite3_enable_load_extension(db_, 1);
05:37:49         ^
05:37:49 plugins/input/sqlite/sqlite_connection.hpp:172:22: error: use of undeclared identifier 'sqlite3_load_extension'
05:37:49         int result = sqlite3_load_extension(db_, ext_path.c_str(), 0 , 0);
05:37:49                      ^
```

**Comment by fxcoudert** [comment]
`mapnik` should be fixed by new version: https://github.com/Homebrew/homebrew-core/pull/33077

File: Formula/icu4c.rb [hunk 1]
```diff
@@ -1,10 +1,10 @@
 class Icu4c < Formula
   desc "C/C++ and Java libraries for Unicode and globalization"
   homepage "https://ssl.icu-project.org/"
-  url "https://ssl.icu-project.org/files/icu4c/62.1/icu4c-62_1-src.tgz"
-  mirror "https://downloads.sourceforge.net/project/icu/ICU4C/62.1/icu4c-62_1-src.tgz"
-  version "62.1"
-  sha256 "3dd9868d666350dda66a6e305eecde9d479fb70b30d5b55d78a1deffb97d5aa3"
+  url "https://ssl.icu-project.org/files/icu4c/63.1/icu4c-63_1-src.tgz"
+  mirror "https://downloads.sourceforge.net/project/icu/ICU4C/63.1/icu4c-63_1-src.tgz"
```
Comment (srl295) [inline Formula/icu4c.rb]: you can also get it from https://github.com/unicode-org/icu/releases/tag/release-63-1 now !

**Comment by SMillerDev** [comment]
@lembacon can you rebase this?

**Comment by lembacon** [comment]
Rebased.

**Comment by lembacon** [comment]
Also switched the mirror from SourceForge to GitHub as @srl295 [pointed out](
https://github.com/Homebrew/homebrew-core/pull/33011#discussion_r227596576).

**Comment by SMillerDev** [comment]
@lembacon can you rebase this (again)? We should really try to get this through quicker.

**Comment by srl295** [comment]
@SMillerDev if there's anything I can do let me know.

**Comment by fxcoudert** [comment]
I'm trying to take over with a clean version at #34533
Wish me luck!

**Pull request closed without merging**